docs/src/pages/components/slider/CustomizedSlider.js

```diff
@@ -48,20 +50,19 @@ const marks = [
   },
 ];
 
-const IOSSlider = withStyles({
-  root: {
-    color: '#3880ff',
-    height: 2,
-    padding: '15px 0',
-  },
-  thumb: {
+const IosSlider = styled(Slider)({
+  color: '#3880ff',
+  height: 2,
+  padding: '15px 0',
+  '& .MuiSlider-thumb': {
     height: 28,
     width: 28,
     backgroundColor: '#fff',
+    // @ts-ignore
```

Comment:
@mnajdova what problem did you have here? how we can solve this?

Reply:
It's craft, removed!

But the scenario where I needed it was the following:
```
styled(Slider)(props => {
  background: 'grey', // default value
  ...(props.color === 'primary' && {
    background: 'blue',
  }),
})
```

emotion was complaining that I am redefining that prop